addons/google_calendar/tests/test_sync_odoo2google.py

```diff
@@ -85,7 +85,7 @@ def test_event_creation_perf(self):
 
             events._sync_odoo2google(self.google_service)
 
-        with self.assertQueryCount(__system__=129):
+        with self.assertQueryCount(__system__=131):
```

Comment:
Note to reader: 2 extra queries added by ORM "select id from notif where msg_id in" when deleting a message due to the addition of a related on it, to correctly update the related field.

They could be removed in this particular case because `mail_message_id` has `ondelete cascade` which means the related record is going to be deleted just afterwards. If we want to remove those 2 queries this is a change that might be possible in the ORM, but it is out of scope of current PR.